packages/pgx_lwt/pgx_lwt.0.1/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+authors: ["Arena Developers <silver-snakes@arena.io>"]
+maintainer: "silver-snakes@arena.io"
+homepage: "https://github.com/arenadotio/pgx"
+dev-repo: "https://github.com/arenadotio/pgx.git"
+bug-reports: "https://github.com/arenadotio/pgx/issues"
+doc: "https://arenadotio.github.io/pgx/doc"
+
+build: ["jbuilder" "build" "-p" "pgx,"name "-j" jobs]
```

Comment:
The `"pgx,"name` looks rather wrong. Why do you need to build pgx if it's already pulled as a dependency ?

(same for the other packages)

Reply:
that's because you are trying that locally where it seems that the `pgx` package is not installed.

`-p` tell you: "I want every dependencies to come from globally installed packages instead of local ones